services/web-app/data/sort.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+export const ALPHABETICAL_ORDER = 'a-z'
+export const ORDER_BY_STATUS = 'status'
+export const sortItems = [
+  {
+    id: ALPHABETICAL_ORDER,
+    text: 'A–Z'
+  },
+  {
+    id: ORDER_BY_STATUS,
+    text: 'Status'
+  }
+]
```

Comment:
It feels a little weird putting this constants file in a data folder in the project. Is there any better home for constants-type stuff in the web-app? (same comment for the view.js file below)